feg/gateway/docker/go/Dockerfile

```
@@ -65,9 +65,10 @@ ENV GOPROXY https://proxy.golang.org
 
 RUN printenv > /etc/environment
 
-COPY feg/radius/lib/go/ /radius/lib/go
-COPY feg/radius/src/go.* /radius/src/
-WORKDIR /radius/src
+COPY feg/radius/lib/go/ /feg/radius/lib/go
+COPY feg/radius/src/go.* /feg/radius/src/
+COPY orc8r/lib/go/go.* /orc8r/lib/go/
```

to align with the code below:
```suggestion
COPY orc8r/lib/go/go.* $MAGMA_ROOT/orc8r/lib/go/
```